packages/flutter/lib/src/widgets/_web_image_web.dart

```diff
@@ -366,3 +390,20 @@ class RenderWebImage extends RenderShiftedBox {
     );
   }
 }
+
+extension on BoxFit {
+  web.CSSObjectFit get cssObjectFit {
+    switch (this) {
```

Comment:
Not enforced but the pattern matching statement is the fashion for you to consider:
```dart
  return switch(this) {
    BoxFit.fill => web.CSSObjectFit.fill,
    ...
  };
```